app/post_metadata.go

```diff
@@ -443,16 +444,42 @@ func (a *App) getCustomEmojisForPost(c request.CTX, post *model.Post, reactions
 }
 
 func (a *App) isLinkAllowedForPreview(link string) bool {
-	domains := a.normalizeDomains(*a.Config().ServiceSettings.RestrictLinkPreviews)
+	domains := normalizeDomains(*a.Config().ServiceSettings.RestrictLinkPreviews)
```

Comment:
How worse would it look if this function explicitly returned `true` and otherwise defaulted to `false`?

Reply:
Can you clarify what are you envisioning? I am failing to see how `normalizeDomains` returns true/false

Reply:
I am referring `isLinkAllowedForPreview`. Essentially changing the function from defaulting to a true return value to a false return value.

Reply:
Ah okay, I'll need to see, but the err != nil checks will have to be there anyways. I am not seeing a big benefit here either ways.

Reply:
Yes, logically it would be the same, it's mostly a good practice with security sensitive functions to make it less likely to introduce a false negative if we were to touch this again in the future. But again, if it turns out to be more complex/less legible I am totally okay with keeping it as is, just curious :)

Reply:
The code becomes counter-intuitive, leaving it as is.